Makefile

```diff
@@ -85,13 +85,13 @@ build-windows-x86:
 install-tools:
 	go install github.com/client9/misspell/cmd/misspell@v0.3.4
 	go install github.com/google/addlicense@v1.1.0
-	go install github.com/goreleaser/goreleaser@v1.16.1
-	go install github.com/mgechev/revive@v1.2.3
+	go install github.com/goreleaser/goreleaser@v1.18.2
+	go install github.com/mgechev/revive@v1.3.1
```

Comment:
I think the github actions need to be updated to use these versions of the tools.

Reply:
I don't see revive in the actions

Reply:
Yeah looks like we just use their action. I double checked. Thanks for updating goreleaser.